docker/brew.Dockerfile

```diff
@@ -128,6 +132,11 @@ RUN $YARN install --frozen-lockfile --production --network-timeout 600000
 FROM registry.redhat.io/ubi9/nodejs-18-minimal:1 AS runner
 USER 0
 
+RUN microdnf update -y && \
+  microdnf install -y python3 python3-pip && \
+  pip3 install mkdocs-techdocs-core==1.* && \
+  microdnf clean all
+
 # Downstream only - do not install techdocs dependencies (not required)
```

Comment:
I guess we can remove this comment since we ARE installing the techdocs reqs now.